include/core/Keyboard.hpp

```diff
@@ -12,10 +12,12 @@
  * =====================================================================================
  */
 #ifndef KEYBOARD_HPP_
-#define KEYBOARD_HPP_
-
+#define KEYBOARD_HPP_
+

```

Comment:
I don't understand why something changed here?

Reply:
It's a line ending thing (CRLF/LF). Idk why it suddenly did that... which line endings should I use?

Reply:
Ok, I'll have to figure out how to convert the wrong ones now

Reply:
If you need help, you can find me on [OpenMiner discord server](https://discord.gg/eN8k8wt) or on irc.freenode.net `#openminer`

Reply:
I didn't setup Git properly, I hope I fixed it now.
Ok, I'll join the discord server when I'm back at my normal PC!